client/src/components/screens/Patient/components/PrescriptionsTab/components/Prescriptions.tsx

```diff
@@ -117,8 +119,8 @@ const Prescriptions: React.FC<Props> = ({ prescriptions, clinicalImpressions })
   }, []);
 
   const practitionerPopOverText = (info: any) => {
-    const phonePart = info.phone.split(' ');
-    const phone = `(${phonePart[0]}) ${phonePart[1]}- ${phonePart[2]} `;
+    const phonePart = split(info.phone, ' ');
```

Comment:
why not using plain javascript vs lodash: something like: (info?.phone || '').split(' ')
then, phonePart.length === 3.

Reply:
lodash can (and should from my point of view) be favored over the default javascript functions as possible.

Reply:
no, if there is native JS support its better to use the native one. It would be faster, reduce bundle size etc. Use lodash when the feature is not standardize accross browser.

```
const phonePart = info?.phone ? info.phone.split(' ') : [];
const phone = phonePart.lentgh === 3 ? `(${phonePart[0]}) ${phonePart[1]}- ${phonePart[2]} ` : info.phone;
```

Reply:
@creativeyann17 I m curious to know why you would favour using a library over native code (in cases when the semantic is pretty much equivalent ex: [].filter() vs []_.filter()). I m sure you have good reasons for your point of view. I'd be glad to hear about it.

Reply:
it's a valid point, I guess it is a bit like politics. However, besides the other points against the use of lodash over native code (which I agree upon), I think that we are "padding/protecting" way to much our variables against null/undefined in the application.  It may not applied to the this pr but for the sake of giving an example imagine that `info`  should never logically be null or undefined. Therefore, by protecting this value we hide a possible bug or a flaw in the way we pass data down to some components. Should it explode to our faces we would know immediately that something's wrong.